config/default/common/config/metadata/stories/fires_overview_2019/step006.md

```diff
@@ -0,0 +1,7 @@
+<p>The near real-time fire information are available through the Fire Information for Resource Management System (FIRMS). FIRMS provides the data for download in easy-to-use file formats, provides the ability for users to set up fire email alerts for their areas-of-interest and provides a fire map with additional information such as burned area.
+</p>
+Additional Information from FIRMS:<br>
+[Download Fire Data](https://firms.modaps.eosdis.nasa.gov/active_fire/#firms-shapefile)<br>
```

Comment:
For list items add a "- " in front of them (unlinked)

Reply:
Was trying to figure out how to do that!